index.html

```diff
@@ -62,17 +62,17 @@
         <div class="row">
           <div class="col-12">
               <nav class="navbar navbar-expand-lg mobile-nav-wrapper">
-                <a class="navbar-brand" href="#">
+                <a class="navbar-brand" href="index.html">
                     <img class="logo-custom-mobile" src="./assets/image/logo.png" alt="" srcset="">
                 </a>
                 <div class="" id="">
                   <ul class="navbar-nav">
                     <li class="nav-item active">
                       <a class="nav-link" href="index.html">Home </a>
                     </li>
```

Comment:
- [ ] On the Home Page of your mobile design, I will suggest that you style the contents of the cards of the `Major courses` section as seen in the Behance Design 👍 
- [ ] I also noticed that your missing the `orange button` underneath the cards so please replace this text `See the whole course` with the `orange button` as seen in the mobile design 😉 
- Your Design
<img width="340" alt="Screen Shot 2023-05-24 at 11 23 52 AM" src="https://github.com/Aqurds/wd-course/assets/102699958/e53c5683-522e-4879-8f35-4c99f9078e2b">

- The behance Design
<img width="388" alt="Screen Shot 2023-05-24 at 11 23 33 AM" src="https://github.com/Aqurds/wd-course/assets/102699958/b5668f17-6761-4fc9-adc9-96c3356a7e4f">

- [ ] I also noticed that the `Main Instructors` section of the mobile view isn't styled as seen in the original design. Kindly fix that as well 👍 
- [ ] I also noticed that the names of the instructors aren't styled as seen in the original design, and the profession of the speakers in the orange color are also missing the Italic effect applied to them as seen in the original design so kindly include this 😉
- [ ] Please do this for both desktop and Mobile 👍 
- Your Design
<img width="286" alt="Screen Shot 2023-05-24 at 11 26 14 AM" src="https://github.com/Aqurds/wd-course/assets/102699958/76983cb0-3608-4795-ae42-a617747b9efb">
- The Behance Design
<img width="388" alt="Screen Shot 2023-05-24 at 11 28 00 AM" src="https://github.com/Aqurds/wd-course/assets/102699958/394cb3ef-766d-4529-8d0a-1f4046a17b8f">

- [ ] The Partners section on the mobile view takes over the entire page. This is not an ideal design and it also looks different from the original design. So kindly ensure that your partners section is smaller and styled as seen in the original design 💯 
- Your Design 
<img width="279" alt="Screen Shot 2023-05-24 at 11 30 28 AM" src="https://github.com/Aqurds/wd-course/assets/102699958/2a4c3364-098d-48ce-82d4-4cf16a299e56">
- The Behance Design
<img width="389" alt="Screen Shot 2023-05-24 at 11 32 57 AM" src="https://github.com/Aqurds/wd-course/assets/102699958/1d43deeb-10cf-44ad-a45e-968b099ed96c">